src/AST/Solved.hs

```diff
@@ -17,7 +18,7 @@ data AST =
   AST
     { aimports   :: [Import]
     , aexps      :: [Exp]
-    , aadts      :: [ADT]
+    , atypedecls :: [TypeDecl]
```

Comment:
That was renamed as now it contains different types of type declarations: adts and aliases.

Reply:
I think it's much clearer now.